ui/admin/client/scripts/pages/execution-metrics.js

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-'use strict';
+('use strict');
```

Comment:
It seems that this was probably caused by a linter? 
It should stay without the brackets. https://www.w3schools.com/js/js_strict.asp